### ISSUE TITLE
Horizon Zero Dawn POPCNT

### ISSUE THREAD

**Issue by milosbelic** [ogurets/popcnt_emulator#9]
Horizon Zero Dawn is a new game with this problem. Emulating SDE is working but with a huge performance hit. Any help?

**Comment by mirh** [comment]
If it's just POPCNT (and not also CRC32) then this should work?

**Comment by milosbelic** [comment]
The thing is I don't understand what I should do with these files you posted. Is there any ELI5 tutorial?

**Comment by mirh** [comment]
I didn't post any file?
Just follow the instructions in the readme and replace QuantumBreak.exe with horizon's.

**Comment by mirh** [comment]
Patch 1.08 removed the AVX requirement. 
Are your sure that couldn't have been it?

**Comment by milosbelic** [comment]
Yeah I saw that, but didn't AVX just replace SSE 4.1/2? The thing is my cpu don't support AVX too. I'm out of luck and it seems that every new game require these instructions.

**Comment by milosbelic** [comment]
Wait, you may be right. I will try it again!
